frontend/src/app/shared/components/dynamic-forms/components/dynamic-form/dynamic-form.component.ts

```diff
@@ -379,6 +380,9 @@ export class DynamicFormComponent extends UntilDestroyedMixin implements OnChang
       fields = this.dynamicFieldsService.getFormlyFormWithFieldGroups(this.fieldGroups, fields);
     }
 
+    const id = this.resourceId || idFromLink(this.resourcePath || null);
+    model.id = id;
```

Comment:
Maybe an explanatory comment is needed good here; the form does not give us an ID because it's not editable, but some fields need IDs to check permissions and/or relations.